pkg/framework/version_bundle.go

```diff
@@ -107,7 +109,7 @@ func checkType(vType string) error {
 		}
 	}
 	if !isValid {
-		return fmt.Errorf("%q is not a valid version bundle version type", vType)
+		return fmt.Errorf("'%s' is not a valid version bundle version type", vType)
```

Comment:
Nitpick but I think this is fine with the `%q`.  We use it in lots of places and I find it easy to read.

Reply:
With my change the first line is more readable IMO than the double quoted one, because this is the rendered result we really see. 

```
"'%s' is not a valid version bundle version type"
"\"%s\" is not a valid version bundle version type"
```

Reply:
OK fine thanks for the explanation.